blockchain/v2/processor.go

```diff
@@ -4,48 +4,32 @@ import (
 	"fmt"
 
 	"github.com/tendermint/tendermint/p2p"
-	tdState "github.com/tendermint/tendermint/state"
+	tmState "github.com/tendermint/tendermint/state"
 	"github.com/tendermint/tendermint/types"
 )
 
-type peerError struct {
-	priorityHigh
-	peerID p2p.ID
-}
-
-type pcDuplicateBlock struct {
-	priorityNormal
-}
-
-type pcShortBlock struct {
-	priorityNormal
-}
-
+// Events generated by the processor:
+// block execution failure, event will indicate the peer(s) that caused the error
 type pcBlockVerificationFailure struct {
 	priorityNormal
 	height       int64
 	firstPeerID  p2p.ID
 	secondPeerID p2p.ID
 }
 
+// successful block execution
 type pcBlockProcessed struct {
 	priorityNormal
 	height int64
 	peerID p2p.ID
 }
 
-type pcProcessBlock struct {
-	priorityNormal
-}
-
-type pcStop struct {
-	priorityNormal
-}
-
+// processor has finished
 type pcFinished struct {
 	priorityNormal
 	height       int64
-	blocksSynced int64
+	blocksSynced int
```

Comment:
do we want to be more strict with int types? is there a reason why this was transitioned from int64 to int?

Reply:
Agreed. Made an issue here #4416